### DISCUSSION TITLE
Disable swap

### DISCUSSION THREAD

**Discussion by alveare1** [Security-Onion-Solutions/securityonion#12995] [2.4]
### Version

2.4.60

### Installation Method

Security Onion ISO image

### Description

configuration

### Installation Type

Distributed

### Location

on-prem with Internet access

### Hardware Specs

Exceeds minimum requirements

### CPU

8 to 16 depending on node

### RAM

16 or 32gb

### Storage for /

512

### Storage for /nsm

512

### Network Traffic Collection

other (please provide detail below)

### Network Traffic Speeds

Less than 1Gbps

### Status

Yes, all services on all nodes are running OK

### Salt Status

No, there are no failures

### Logs

No, there are no additional clues

### Detail

In the instructions it says "If you have plenty of RAM, disable swap altogether." 

It doesn't say how. Do I need to do anything special?

### Guidelines

- [X] I have read the discussion guidelines at https://github.com/Security-Onion-Solutions/securityonion/discussions/1720 and assert that I have followed the guidelines.

**Comment by dougburks** [comment]
https://www.elastic.co/guide/en/elasticsearch/reference/current/setup-configuration-memory.html#disable-swap-files
